ptbr/training_cli.py

```
@@ -0,0 +1,934 @@
+        yaml.dump(cfg, f, default_flow_style=False, sort_keys=False)
+    logger.info(f"Resolved config saved to {resolved_path}")
+
+    # ---- Launch training ----
+    _launch_training(cfg, output_folder, resume=resume)
+
+
+# ======================================================================== #
+#  TRAINING LAUNCHER                                                        #
+# ======================================================================== #
+
+def _launch_training(cfg: dict, output_folder: Path, resume: bool) -> None:
```

Ruff (0.15.0)

[warning] Unused function argument: `resume`

(ARG001)